MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs

```diff
@@ -530,7 +532,7 @@ public virtual Task<bool> CloseMasterDetailPage(IMvxViewModel viewModel, MvxMast
                 case MasterDetailPosition.Master:
                     return ClosePage(masterDetailHost.Master, null, attribute);
```

Comment:
Shouldn't we do the same for the master? In theory it could have navigation too

Reply:
I'm not sure since the current behavior for `MasterDetailPosition.Root` and `MasterDetailPosition.Master` is to send a root page which is then ignored by `ClosePage` method and will try to `PopAsync` from the navigation stack but `master detail` as far as I know it doesn't push itself into the navigation stack, knowing that I don't think we can do `popAsync or removePage` from the navigation stack?

Reply:
Might it require a different implementation?